src/Phantoman.php

```diff
@@ -9,75 +12,102 @@
+use Codeception\Events;
 use Codeception\Exception\ExtensionException;
+use Codeception\Extension;
 
-class Phantoman extends \Codeception\Platform\Extension
+/**
+ * Class Phantoman.
+ *
+ * @package Codeception\Extension
+ */
+class Phantoman extends Extension
 {
-    // list events to listen to
-    static $events = array(
-        'module.init' => 'moduleInit',
-    );
 
+    /**
+     * Events to listen to.
+     *
+     * @var array
+     */
+    protected static $events = [
+        Events::SUITE_INIT => 'suiteInit',
+    ];
+
+    /**
+     * A resource representing the PhantomJS process.
+     *
+     * @var resource
+     */
     private $resource;
 
+    /**
+     * File pointers that correspond to PHP's end of any pipes that are created.
+     *
+     * @var array
+     */
     private $pipes;
 
-    public function __construct($config, $options)
+    /**
+     * Phantoman constructor.
+     *
+     * @param array $config
+     *   Current extension configuration.
+     * @param array $options
+     *   Passed running options.
+     *
+     * @throws \Codeception\Exception\ExtensionException
+     */
+    public function __construct(array $config, array $options)
     {
-        // Codeception has an option called silent, which suppresses the console output.
-        // Unfortunately there is no builtin way to activate this mode for a single extension.
-        // This is why the option will passed from the extension configuration ($config)
-        // to the global configuration ($options);
+        // Codeception has an option called silent, which suppresses the console
+        // output. Unfortunately there is no builtin way to activate this mode for
+        // a single extension. This is why the option will passed from the
+        // extension configuration ($config) to the global configuration ($options);
         // Note: This must be done before calling the parent constructor.
         if (isset($config['silent']) && $config['silent']) {
             $options['silent'] = true;
         }
-
         parent::__construct($config, $options);
 
-        // Set default path for PhantomJS to "vendor/bin/phantomjs" for if it was installed via composer
+        // Set default path for PhantomJS to "vendor/bin/phantomjs" for if it was
+        // installed via composer.
         if (!isset($this->config['path'])) {
             $this->config['path'] = 'vendor/bin/phantomjs';
         }
 
-        // If a directory was provided for the path, use old method of appending PhantomJS
-        if (is_dir(realpath($this->config['path']))) {
-            // Show warning that this is being deprecated
-            $this->writeln(PHP_EOL);
-            $this->writeln('WARNING: The PhantomJS path for Phantoman is set to a directory, this is being deprecated in the future. Please update your Phantoman configuration to be the full path to PhantomJS.');
-
-            $this->config['path'] .= '/phantomjs';
-        }
```

Comment:
With this removal, I think I'm going to have to bump Phantoman up to version 2.0. I have no problem with that, just making a note of it 😄